consensus/istanbul/core/core.go

```diff
@@ -353,42 +353,51 @@ func (c *core) stopTimer() {
 func (c *core) newRoundChangeTimer() {
 	c.stopTimer()
 
+	// TODO-Klaytn-Istanbul: Replace &istanbul.DefaultConfig.Timeout to c.config.Timeout
```

Comment:
For now, it is impossible to change this value by governacne? @aidan-kwon

Reply:
A governance item for Istanbul timeout is prepared now, but I think it is not working now. 
We should finish the implementation later.